examples/kubernetes/connectivity-check/connectivity-check.yaml

```diff
@@ -73,10 +73,62 @@ spec:
       - name: echo-container
         image: docker.io/cilium/json-mock:1.0
         imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 80
+          hostPort: 40000
```

Comment:
Also added one here for services (https://github.com/cilium/cilium/commit/ebd73929c3ae97a7a37424b10c603cd87d591bc8) but only for the BPF case so far. I wonder if we somewhat could consolidate both, just a thought; not needed for this series in here.